.storybook/stories/KitchenSink.tsx

```diff
@@ -443,16 +443,18 @@ function KitchenSink({ background }: KitchenSinkProps) {
 
 							<CallToActionLink href="#">Call to action</CallToActionLink>
 
-							<DropdownMenu>
-								<DropdownMenuButton>Open dropdown menu</DropdownMenuButton>
-								<DropdownMenuPanel>
-									<DropdownMenuItem>Profile</DropdownMenuItem>
-									<DropdownMenuItem>Messages</DropdownMenuItem>
-									<DropdownMenuItem>Account settings</DropdownMenuItem>
-									<DropdownMenuDivider />
-									<DropdownMenuItem>Sign out</DropdownMenuItem>
-								</DropdownMenuPanel>
-							</DropdownMenu>
+							<div>
```

Comment:
Wrapping the `DropdownMenu` here as the there is a `Stack` wrapping these components which is causing the button to go full-width